pages/installation.md

```diff
@@ -24,34 +24,41 @@ Caveats with the following versions of Elixir and Erlang are documented below:
 |  25         | `>= 25.0`        |        |
 
 ## Prerequisites
-First, Install git LFS by [following these instructions](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage).
 
-Next, [install `just`](https://github.com/casey/just?tab=readme-ov-file#cross-platform)
+Before building the project, make sure you have the following tools installed:
 
-Next, [install `zig`](https://ziglang.org/learn/getting-started/) if not already installed. **Important:** version 0.14.1 is required, which is currently _not_ the latest version.
+### 1. Git LFS
```

Comment:
Is this required? I dont ever remember installing git lfs

Reply:
This part of the docs is carried over from lexical, though I think it's not necessary for us